src/platforms/common/enriching-events/screenshots.mdx

```diff
@@ -15,12 +17,6 @@ When a user experiences an error, Sentry provides the ability to take a screensh
 
 This feature only applies to SDKs with a user interface, such as the ones for mobile and desktop applications. In some environments like native iOS, taking a screenshot requires the UI thread and in the event of a crash, that might not be available. Another example where a screenshot might not be available is when the event happens before the screen starts to load. So inherently, this feature will be a best effort solution.
 
-<PlatformSection supported={["apple.ios"]}>
-
-On iOS, only manually captured errors or exceptions will contain a screenshot.
```

Comment:
`m`: I think we should keep this comment and mention that since 7.20.0 crashes also have screenshots.

Reply:
Agreed, can be done on a follow up PR since the blog post is live: https://blog.sentry.io/2022/07/07/introducing-mobile-screenshots-and-suspect-commits/